ng2-components/ng2-alfresco-core/src/components/collapsable/accordion-group.component.spec.ts

```diff
@@ -56,6 +56,28 @@ describe('AccordionGroupComponent', () => {
         });
     });
 
+    it('should hide expand icon by default', () => {
+        component.heading = 'Fake Header';
+        component.headingIcon = 'fake-icon';
+        component.contentWrapper.nativeElement.innerHTML = '';
+        fixture.whenStable().then(() => {
+            fixture.detectChanges();
+            let headerToggle = fixture.nativeElement.querySelector('.adf-panel-heading-toggle .material-icons');
```

Comment:
please use the id of the templates to use the queryselector. Don't refer to the class it can change  easly and will make the test failing . If you don't have the ID add it